### PR TITLE
Events display working group project etc

### PR DESCRIPTION
**Pull request by markcsims**
- [x] Display WorkingGroup/Project/GP2 Hub on event summary
- [x] Display WorkingGroup/Project/GP2 Hub on event detail
- [x]  fixes speaker count when there are no speakers

### CODE REVIEW

File: packages/gp2-components/src/molecules/EventOwner.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { projectIcon, workingGroupIcon } from '../icons';
+import IconWithLabel from './IconWithLabel';
+
+const { rem } = pixels;
+
+const { projects, workingGroups } = gp2Routing;
+const listItemStyles = css({
+  padding: `${rem(7.5)} 0`,
+  color: neutral900.rgb,
+  whiteSpace: 'break-spaces',
```
Comment (jhonny-m) [inline packages/gp2-components/src/molecules/EventOwner.tsx:14]: do we need this styling?

Comment (jhonny-m) [reply packages/gp2-components/src/molecules/EventOwner.tsx:14]: whiteSpace, overflow, textOverflow and fontSize that is

File: packages/gp2-components/src/molecules/EventOwner.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import { projectIcon, workingGroupIcon } from '../icons';
+import IconWithLabel from './IconWithLabel';
+
+const { rem } = pixels;
+
+const { projects, workingGroups } = gp2Routing;
+const listItemStyles = css({
+  padding: `${rem(7.5)} 0`,
```
Comment (jhonny-m) [inline packages/gp2-components/src/molecules/EventOwner.tsx:12]: I think the IconWithLabel has this margin, we are removing it then adding it back here has a padding

File: packages/gp2-components/src/molecules/EventOwner.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    project && {
+      ...project,
+      href: projects({}).project({ projectId: project.id }).$,
+      icon: projectIcon,
+    }
+  );
+};
+
+const EventOwner: React.FC<EventOwnerProps> = ({ project, workingGroup }) => {
+  const { title, href, icon } = getProperties({ project, workingGroup }) || {
```
Comment (jhonny-m) [inline packages/gp2-components/src/molecules/EventOwner.tsx:42]: getProperties can check and return the default props, we don't have to do it here

File: packages/gp2-components/src/molecules/EventOwner.tsx (hunk 2 above)
Comment (markcsims) [reply packages/gp2-components/src/molecules/EventOwner.tsx:12]: done

File: packages/gp2-components/src/molecules/EventOwner.tsx (hunk 3 above)
Comment (markcsims) [reply packages/gp2-components/src/molecules/EventOwner.tsx]: done

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**